liquid-compiler/src/registry.rs

```diff
@@ -73,7 +73,7 @@ macro_rules! delegate_iterator {
             type Item = $item;
             #[inline]
             fn next(&mut self) -> Option<Self::Item> {
-                self.iter.next().map(|s| *s)
```

Comment:
Allocating on iteration is a bad practice.  We should be able to get away with `&str`